### ISSUE TITLE
Location service does not start by itself in LOS 17.1 (Android 10)

### ISSUE THREAD

**Issue by jw243** [microg/android_packages_apps_GmsCore#1018]
I am using LOS 17.1 (Android 10) on an Nexus 7 (2013) aka **flo**. MicroG components were installed as system apps using Magisk module "microG Installer Revived".

When I configure UnifiedNlp using Mozilla Location Service, location detection via Wifi works as expected. However, after not using any location app for a while respectively after a reboot location detection does not work anymore. It starts to work again, when I manually start Mozilla Location Service by going back into its settings. For some reason, Mozilla Location Service does not start by itself.

One thing I noticed: Location permission is allowed to microG **only while in use** instead of **all the time**. This cannot be changed in the settings. Could this be the reason for this strange behaviour? Maybe the manifest of MicroG Service Core needs to be updated such that location access is allowed all the time?

Any help would  be greatly appreciated.

**Comment by ArchangeGabriel** [comment]
Yes, see https://github.com/microg/android_packages_apps_GmsCore/issues/986.

**Comment by jw243** [comment]
I just made an interesting observation:

After reboot, it seems as if Mozilla UnifiedNlp does not start by itself. It has to be started manually by tapping in microG Settings > UnifiedNlp Settings > Configure location backends > OK.

If you **disable battery optimisation** for Mozilla UnifiedNlp Backend, then location detection works instantaneously in SatStat or OsmAnd until the next reboot!
